requirements.txt

```diff
@@ -1,15 +0,0 @@
-Django<2.1
-psycopg2
-psycopg2-binary
-Pillow
-gunicorn
-djangorestframework
-celery
-vendor/django-registration.zip
-Markdown
-coreapi
-confusable-homoglyphs>=3.1.1
-django-machina
-django-haystack
-Whoosh
--r requirements_importer.txt
```

Comment:
Because of incorrect architecture, the concordia app has to install the requirements for the importer app (for the time being until the architecture is fixed). I do want to keep the importer and concordia requirements separate, but, for now all the requirements have to be installed in both docker containers to get it to work (this is one of the major things on my to-do list).

Reply:
We might want to see how Pipfile handles optional dependencies or whether it'd be easier just to have them installed in both places. Wheel caching has made that a lot cheaper than it used to be.